### ISSUE TITLE
Fade out the screen when vote

### ISSUE THREAD

**Issue by armync** [evandrocoan/MultiModServer#81]
- A command to toggle a function like at the final of the round, vote and make the screen black?

With this, this can be avoid:

1) Players that want to change the weapon (ex. knife-pistol-knife) and by mistake they press a button and vote a bad (non-favorite) map. (This happend for me many times)
2) Some players don't want to vote or focus on the vote (because it's during the round) and they ignore it or random vote, so by default if it make the screen black the majority of players will vote their favorite map.

[Video Preview](https://www.youtube.com/watch?v=Ck_NqI4hZmg)
[Plugin with this feature](https://translate.google.com/translate?hl=&sl=auto&tl=en&u=http%3A%2F%2Fnon-steam.ru%2F%3Fp%3D272) (You can use the code or just inspire.. So you can't say that will take more time or something like that)

**Comment by armync** [comment]
**Any release date?**

**Comment by evandrocoan** [comment]
I am not sure, but I think it can be on 2020 or 2050.

**Comment by armync** [comment]
I will still live fortunately but why so? I think that's a good ideea and simple for you to code (and you have that plugin source too). I saw more issues non-fixed. Do you abbandoned this project? So if you didn't liked the ideea you shouldn't delete this?

PS: So I should expect an update like this in a week or two with 0.5% chance 😀?

**Comment by evandrocoan** [comment]
I have a lot of other projects to finish, like enough to keep-me busy for the next 10 or 50 years. Also, I do not run servers or play gamer anymore, neither I am planing to do. 

Why I would delay the other project I am needing, to do this project which I do not need?

**Comment by armync** [comment]
> "..."

Yeah I understand.. I said this because I saw that you still update it and I saw something about you can continue it in January.

Thanks for the reply.
